modules/sandboxed-containers-check-nodes.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying-sandboxed-container-workloads.adoc
+
+:_module-type: PROCEDURE
+[id="sandboxed-containers-check-nodes_{context}"]
+= Checking the eligibility of cluster nodes to run Kata containers
+
+You can check whether it is possible to run sandboxed containers by checking for the `feature.node.kubernetes.io/runtime.kata` label on nodes. Any node where this label is present can run Kata containers.
```

Comment:
@aireilly With respect to the three options I outlined in the comment right above, this is strictly speaking only true for the last one - if the user installs NFD, then they can check node eligibility by checking the label since the eligibility evaluation was basically done for them by NFD (in cooperation with OSC).

For the second option above, well, they can still check node eligibility by checking the label but under these circumstances they'd be just checking information that they themselves set up.

In case of the first option above, they cannot check node eligibility by checking the label.  Since there's nobody to create the label under this scenario (no NFD and no manual user intervention), nodes without the label can very well be eligible to run kata.

I understand this might be getting confusing, if in doubt please don't hesitate to ask for additional explanation/confirmation! :-)

Reply:
I was overthinking it. Thank you for the explanation.